javascript/javascript1/week2/fullname.js

```diff
@@ -0,0 +1,38 @@
+function getFullname(firstname,surname){
+        return firstname+" "+surname;
+    
+}
+const fullname1 = getFullname("Benjamin","Huges");
+const fullname2 = getFullname("Henrik","Olsen");
+console.log(fullname1);
+console.log(fullname2);
+
+//Formal Fullname
+function getFullname(firstname, surname,useFormalName,gender){
```

Comment:
Don't create multiple functions with the same name.
The last one will override all functions before.
Keep only one that includes all conditions and remove the rest.